scripts/backup

```diff
@@ -6,24 +6,24 @@
 # IMPORT GENERIC HELPERS
 #=================================================
 
-if [ ! -e _common.sh ]; then
-	# Get the _common.sh file if it's not in the current directory
-	cp ../settings/scripts/_common.sh ./_common.sh
-	chmod a+rx _common.sh
-fi
-source _common.sh
+source ../settings/scripts/_common.sh
 source /usr/share/yunohost/helpers
 
 #=================================================
 # MANAGE SCRIPT FAILURE
 #=================================================
 
+ynh_clean_setup () {
+	### Remove this function if there's nothing to clean before calling the remove script.
+	true
+}
```

Comment:
As the comment says.

Reply:
Kept for future improvement, for example when experimental helper [ynh_systemd_action](https://github.com/YunoHost-Apps/Experimental_helpers/tree/master/ynh_systemd_action) will become official, we will need that step.

Also very interesting step when developing or upgrading a package, I usually add `read -p "Press any key..."` before the true. In case of error during the installation, the process wait for something press on the keyboard to uninstall... So be able to be in the exact situation of the installation error and make some tests to solve and fix the error

Reply:
We won't use `ynh_systemd_action` for this app, there's no specific service to start.

Agree that it can be interesting to have `read` before the remove.
I'd rather prefer to have that included in the new debug helper or to be added when needed.
But as for comments, I'd rather prefer to not keep unused part of the code in an official app.

Reply:
If you agree, I will directly make those changes in the PR:
- removing unneeded parts
- removing unneeded comments